### ISSUE TITLE
Make preview window scrollable

### ISSUE THREAD

**Issue by cloggier** [npxbr/glow.nvim#7]
It seems like `:Glow` floating window only shows end of rendered output (depending on window size) and does not allow to scroll its content as pressing any button just closes the window (it always prints `[Process exited 0]` at the bottom of floating window, which also could be hidden if no errors were detected).

Looks very promising though, will be happy to replace [iamcco/markdown-preview.nvim](https://github.com/iamcco/markdown-preview.nvim).

**Comment by ellisonleao** [comment]
Hey @cloggier thanks for the suggestion. the `[Process exited 0]` msg is a little bit trickier to remove because i am using a neovim terminal to show the output and i haven't found an easy way to just remove that msg. If you have any suggestions please feel free to open a PR so we can discuss it. 

Abou the scrolling part,  i can see the scroll works on my machine using up and down keys and also j,k . Do you have any way to show some steps to reproduce the issue?

**Comment by fffed** [comment]
@ellisonleao, is it possible to scroll a preview window on an initial render to keep it in sync with the current position of a cursor if we run `:Glow` for the current file?

P.S. thanks for the cool pluggin.

**Comment by ahmedelgabri** [comment]
`glow` has a `--pager` flag but the downside of using that is that you will need to do `q` to exit the pager then close the floating window. So two steps instead of one, in order to exit. Maybe can be an option in the plugin?

**Comment by ellisonleao** [comment]
> is it possible to scroll a preview window on an initial render to keep it in sync with the current position of a cursor if we run :Glow for the current file?

yeah, we can definitely grab the position and move the preview window buffer to the same line.

**Comment by ellisonleao** [comment]
> glow has a --pager flag but the downside of using that is that you will need to do q to exit the pager then close the floating window. So two steps instead of one, in order to exit. Maybe can be an option in the plugin?

The preview itself should be already scrollable, but i need to check it behaves on small windows, so i don't think the pager is a good default option but can definitely be a flag in the plugin command for sure

**Comment by ahmedelgabri** [comment]
> > glow has a --pager flag but the downside of using that is that you will need to do q to exit the pager then close the floating window. So two steps instead of one, in order to exit. Maybe can be an option in the plugin?
> 
> The preview itself should be already scrollable, but i need to check it behaves on small windows, so i don't think the pager is a good default option but can definitely be a flag in the plugin command for sure

Yeah, I understand that, but I guess didn't read the main issue properly. @cloggier maybe you need to check your mappings. Because by default it the floating window buffer should be scrollable. 

@ellisonleao can we make sure that when the buffer opens that we are always on the top? currently, it jumps to the bottom. So maybe after opening the buffer the plugin sends `gg` to the buffer?

**Comment by ellisonleao** [comment]
> currently, it jumps to the bottom

thats probably a bug..

> So maybe after opening the buffer the plugin sends gg to the buffer?

I think for a good user experience it would be nice to put the cursor in the same line as the original buffer. Shouldnt be too hard to make it happen

**Comment by ellisonleao** [comment]
closing this since its not an issue. For improvements or suggestions, let's create another ones

**Comment by FilBot3** [comment]
For me changing Glow's config to `pager: false` made ti so I could scroll in Neovim's window. Just wish i could put it in a second window and do it while editing.

**Comment by babygau** [comment]
@ellisonleao I got the same issue, markdown preview is not scrollable, typing any keys make preview window quit. 
FYI, I'm using
```
:version
NVIM v0.6.0-dev
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by nixbld1

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/nix/store/h2fxbg4yjwza4q9shz9k4rzr5ryrrmvq-neovim-unwrapped-master/share/nvim"

Run :checkhealth for more info
```

In cli, I also gotta use `glow -p path/to/md` to be able to scroll.

**Comment by karlredman** [comment]
@babygau / @ellisonleao I am running into the exact same thing -regardless of terminal and even isolated as glow being the only plugin. I can provide more info upon request -if the ticket gets readressed.
